magento/magento2
# Ignore lib/internal/Magento/Framework/App/Test/Unit/_files/app/etc/en…

…v.php that gets generated by running unit tests

### Description (*)
When running Magento Framework unit tests on your local, one of the tests (don't ask me which one) generates a file: `lib/internal/Magento/Framework/App/Test/Unit/_files/app/etc/env.php`
So I think we should ignore this file in git.

### Related Pull Requests

### Fixed Issues (if relevant)
None

### Manual testing scenarios (*)
1. Clone this repo
2. Check that your git status isn't dirty
3. Run `vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist lib/internal/Magento/Framework/`
4. When it's finished, check your git status again
5. Expected is that it's still not dirty, but before this PR, it will show a new file found: `lib/internal/Magento/Framework/App/Test/Unit/_files/app/etc/env.php`

### Questions or comments
In my opinion, it's not needed to run automatic tests on this PR, as it shouldn't affect anything besides local developer experience...

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#39304: Ignore lib/internal/Magento/Framework/App/Test/Unit/_files/app/etc/en…
